.editorconfig

```diff
@@ -289,3 +289,9 @@ dotnet_diagnostic.CA1827.severity = warning
 
 # CA1822: This complains to make functions static. Static is evil. Don't let it complain.
 dotnet_diagnostic.CA1822.severity = none
+
+# Spell Checker rules
+spelling_languages = en-us
+spelling_checkable_types = strings,identifiers,comments
+spelling_error_severity = information
+spelling_exclusion_path = .\exclusion.dic
```

Comment:
What if it would be .\exclusion.txt
Can you check if the spellchecker can handle it? 

It would fix the following: 
![image](https://user-images.githubusercontent.com/9844978/227918518-30165bca-71ca-4d18-b917-b0325620cbf9.png)

Reply:
I tried this but it didn't work.

Reply:
It is worth the effort otherwise we won't see the changes in the dictionary file. @yahiheb pls check this to solve the problem https://stackoverflow.com/questions/6855712/why-does-git-treat-this-text-file-as-a-binary-file